SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+private:
+	
+	static bool IsSpatialOffloadingEnabled();
+
+	static class UActorGroupManager* GetActorGroupManager(const UObject* WorldContext);
+
+public:
+
+	/**
+	 * Returns true if SpatialOS Networking is enabled.
+	 */
+	UFUNCTION(BlueprintPure, Category = "SpatialOS")
+	static bool IsSpatialNetworkingEnabled();
+
+	/**
+     * Returns true if the current Worker Type owns the Actor Group this Actor belongs to.
+	 * Equivalent to HasAuthority when Spatial Networking is disabled.
+     */
+    UFUNCTION(BlueprintPure, Category = "SpatialOS|Offloading")
+    static bool IsActorGroupOwnerForActor(const AActor* Actor);
+
+	/**
+	 * Returns true if the current Worker Type owns the Actor Group this Actor Class belongs to.
+	 * Equivalent to HasAuthority when Spatial Networking is disabled.
```

Comment:
Same comment as I put bellow. I don't think this is true.

Reply:
You're correct, hadn't updated the comment, best equivalent is World->GetNetMode() != NM_Client, which covers Listen Server, Dedicated Server, Standalone.